Plugin: 呆呆兽都会起来锻炼 H2E

**插件名称：**

呆呆兽都会起来锻炼 H2E

**插件功能：**

类似"今天吃什么"的一个插件，通过命令返回健身、游泳、瑜伽的锻炼方式啦

**PyPI 项目名：**

nonebot-plugin-h2e

**插件 import 包名：**

nonebot_plugin_h2e

**插件项目仓库/主页链接：**

https://github.com/Gin2O/nonebot_plugin_h2e

**标签：**

[{"label":"锻炼","color":"#da4a4a"},{"label":"t:what2eat","color":"#99da4a"},{"label":"t: how2exe","color":"#99da4a"}]

# 📃 商店发布检查结果

> Plugin: 呆呆兽都会起来锻炼 H2E

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot-plugin-h2e 的信息如下：
    name         : nonebot-plugin-h2e                            
     version      : 0.1.3                                         
     description  : A nonebot2 plugin package for how to exercise 
    
    dependencies
     - aiohttp &gt;=3.8.4,&lt;4.0.0
     - httpx &gt;=0.23.3,&lt;0.24.0
     - nonebot-adapter-onebot &gt;=2.2.1,&lt;3.0.0
     - nonebot2 &gt;=2.0.0rc3,&lt;3.0.0
插件 nonebot-plugin-h2e 加载出错：
02-23 15:27:19 [SUCCESS] nonebot | NoneBot is initializing...
02-23 15:27:19 [INFO] nonebot | Current Env: prod
02-23 15:27:19 [ERROR] nonebot | Failed to import &quot;nonebot-plugin-h2e&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;nonebot-plugin-h2e&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named &#x27;nonebot-plugin-h2e&#x27;</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 标签: 锻炼-#da4a4a, t:what2eat-#99da4a, t: how2exe-#99da4a。</li><li>✅ 项目 <a href="https://github.com/Gin2O/nonebot_plugin_h2e">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/nonebot-plugin-h2e/">nonebot-plugin-h2e</a> 已发布至 PyPI。</li></code></pre></details>

---

💡 如需修改信息，请直接修改 issue，机器人会自动更新检查结果。
💡 当插件加载测试失败时，请发布新版本后在当前页面下评论任意内容以触发测试。

💪 Powered by [NoneBot2 Publish Bot](https://github.com/nonebot/nonebot2-publish-bot)
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -4436,5 +4436,28 @@
       }
     ],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot_plugin_h2e",
+    "project_link": "nonebot-plugin-h2e",
+    "name": "呆呆兽都会起来锻炼 H2E",
+    "desc": "类似\"今天吃什么\"的一个插件，通过命令返回健身、游泳、瑜伽的锻炼方式啦",
+    "author": "Gin2O",
+    "homepage": "https://github.com/Gin2O/nonebot_plugin_h2e",
+    "tags": [
+      {
+        "label": "锻炼",
+        "color": "#da4a4a"
+      },
+      {
+        "label": "t:what2eat",
+        "color": "#99da4a"
+      },
+      {
+        "label": "t: how2exe",
+        "color": "#99da4a"
+      }
+    ],
+    "is_official": false
   }
 ]

